src/client/pydaos/pydaos_shim.c

```diff
@@ -0,0 +1,217 @@
+/**
+ * (C) Copyright 2019 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+ * The Government's rights to use, modify, reproduce, release, perform, display,
+ * or disclose this software are subject to the terms of the Apache License as
+ * provided in Contract No. B609815.
+ * Any reproduction of computer software, computer software documentation, or
+ * portions thereof marked with this legend must also reproduce the markings.
+ */
+
+#ifdef __USE_PYTHON3__
+#include <python3.6m/Python.h>
```

Comment:
Basically there is no nice way to make this portable to all distros.

SLES-12sp3/Leap 42.3 put the header files in /usr/include/python3.4m/include
The rest are using python3.6m/include as that appears to be the lowest version that the upstream Python is currently supporting for Python 3.

And this code as is is going to break when the distros upgrade their python from 3.6.

So something needs to probe where the python3 Python.h is and pass it as a #define.

https://stackoverflow.com/questions/42858048/how-do-i-use-a-preprocessor-macro-inside-an-include

Reply:
Thanks, resolved with help from Jeff and the team, using ParseConfig("pkg-config --cflags pyhon3") to set the proper include dir.